src/cli/flags.go

```diff
@@ -10,6 +10,7 @@ const (
 	Build      FlagsType = "build"
 	Kubernetes           = "kubernetes"
 	Registry             = "registry"
+	Init       FlagsType = "init"
```

Comment:
do only Build and Init need to have FlagsType explicitly set?

Reply:
I guess we can set in the others as well. let me do it

Reply:
Code updated